src/components/Tutorial/TutorialPlayerInfoPanel.vue

```diff
@@ -79,16 +96,15 @@
           ' one or more stacks of cards in one path that equal up to the group card value. Let\'s protect your true path with'
           + ' a group card. Click on the check boxes above the cards in your true path to group them. You also get a small bonus for using group cards.',
           'You\'re getting closer to completing a path, add the variable (5) card to build up your false path.',
-          'The computer has no grouped stacks, so use the hack card on one of its stacks to set him back.',
+          'The computer has at least one stack that isn\'t grouped, so use the hack card on one of its stacks to set him back.',
           'Your false path is vulnerable to getting hacked, but you dont have any more group card. Use the Firewall card to get full protection from hack cards.',
-          'The computer is vulnerable to getting a virus, send him a virus to slow down his program to half the instructions.',
+          'The computer is vulnerable to getting a virus, send him a virus to slow down his program and half their instructions.',
```

Comment:
"their total instructions" to emphasize that it affects the total, not the cards themselves.